src/mame/vsystem/f1gp.cpp

```diff
@@ -120,6 +124,8 @@ class f1gp_state : public driver_device
 	required_device<palette_device> m_palette;
 	optional_device<generic_latch_8_device> m_soundlatch; // not f1gpbl
 	required_device<acia6850_device> m_acia;
+	optional_device<rs232_port_device> m_rs232a;
+	optional_device<rs232_port_device> m_rs232b;
```

Comment:
Does the bootleg also have the same linking hardware? Then those could be changed to `required_device` and hooked up there too.